docs/create-a-release.md

```diff
@@ -1,13 +1,22 @@
 # Creating a release
 
-1.  Ensure all deps in node_modules installable from npm.
-2.  Update CHANGES.md, with date release, notes, and version.
-3.  Review github issues, close and triage issues related to the release.
-4.  Make sure you have a clean branch: git checkout -b release upstream/master
-5.  Run `npm version v0.0.0 -m "version 0.0.0"` with the version number replacing 0.0.0
-6.  Run `git push upstream master`
-7.  Run `git push upstream --tags`
-8.  Run `npm publish`
-9.  Run `npm tag seneca@VERSION next`
-10. Run `npm tag seneca@VERSION plugin`
-11. Update and publish plugins
+1.  Review github issues, triage, close and merge issues related to the release.
+2.  Navigate to the [Org Test Rig][TestRig] and ensure all test are passing.
+3.  Update CHANGES.md, with date release, notes, and version.
+4.  Pull down the repository locally on the master branch.
+5.  Ensure there are no outstanding commits and the branch is clean.
+6.  Run `npm install` and ensure all dependencies correctly install.
+7.  Run `npm run test` and ensure testing and linting passes.
+8.  Run `npm version vx.x.x -m "version x.x.x"` where `x.x.x` is the version.
+9.  Run `git push upstream master --tags`
+10. Run `npm publish`
+11. Run `npm tag seneca@VERSION next`
+12. Run `npm tag seneca@VERSION plugin`
```

Comment:
I just realize this might create confusion. We should put in docs something like VERSION_NEXT and VERSION_PLUGIN for `next` and `plugins` tags as they are different versions.